classes/Customer.php

```diff
@@ -747,6 +747,14 @@ public static function checkPassword($idCustomer, $passwordHash)
         if (!Validate::isUnsignedId($idCustomer)) {
             die(Tools::displayError());
         }
+
+        // Check that customers password hasn't changed since last login
+        $context = Context::getContext();
+        if ($passwordHash != $context->cookie->__get('passwd')) {
+            $context->customer->logout();
```

Comment:
It seems this line in not needed, as the AdminController is already able to logout the employee if invalid.